plugin-gradle/src/main/java/com/diffplug/gradle/spotless/FormatExtension.java

```diff
@@ -219,32 +219,24 @@ protected final FileCollection parseTarget(Object target) {
 	}
 
 	private final FileCollection parseTargetIsExclude(Object target, boolean isExclude) {
-		if (target instanceof FileCollection) {
+		if (target instanceof Collection) {
+			return parseTargetsIsExclude(((Collection) target).toArray(), isExclude);
+		} else if (target instanceof FileCollection) {
 			return (FileCollection) target;
-		} else if (target instanceof String ||
-				(target instanceof List && ((List<?>) target).stream().allMatch(o -> o instanceof String))) {
+		} else if (target instanceof String) {
 			File dir = getProject().getProjectDir();
-			PatternFilterable userExact; // exactly the collection that the user specified
-			if (target instanceof String) {
-				userExact = getProject().fileTree(dir).include((String) target);
-			} else {
-				// target can only be a List<String> at this point
-				@SuppressWarnings("unchecked")
-				List<String> targetList = (List<String>) target;
-				userExact = getProject().fileTree(dir).include(targetList);
-			}
-			boolean filterOutGitAndGradle;
+			ConfigurableFileTree matchedFiles = getProject().fileTree(dir);
+			String targetString = (String) target;
+			matchedFiles.include(targetString);
+
 			// since people are likely to do '**/*.md', we want to make sure to exclude folders
 			// they don't want to format which will slow down the operation greatly
 			// but we only want to do that if they are *including* - if they are specifying
 			// what they want to exclude, we shouldn't filter at all
-			if (target instanceof String && !isExclude) {
-				String str = (String) target;
-				filterOutGitAndGradle = str.startsWith("**/*") || str.startsWith("**\\*");
-			} else {
-				filterOutGitAndGradle = false;
+			if (isExclude) {
+				return matchedFiles;
 			}
-			if (filterOutGitAndGradle) {
+			if (targetString.startsWith("**/") || targetString.startsWith("**\\")) {
```

Comment:
Just noting that this changes our trigger from `**/*` to `**/`. I think it is a bugfix, but it's worth a mention in the changelog.

Reply:
Done!